code/game/objects/items/storage/belt.dm

```diff
@@ -337,6 +337,12 @@
 	icon_state = "ncr_belt"
 	item_state = "ncr_belt"
 
+/obj/item/storage/belt/military/reconbandolier
```

Comment:
Bandolier might be silly if it can hold boxes of ammo and what not, muh immursions

Reply:
It's like the one ranger jackson wears, not a bunch of bullet loops. It's actually even more plausible for this to hold ammo boxes rather than a duty belt.

Reply:
I do need until tonight to fix something anyway.